sdk/identity/azure-identity/src/main/java/com/azure/identity/AzureCliCredentialBuilder.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity;
+
+/**
+ * Fluent credential builder for instantiating a {@link AzureCliCredential}.
+ *
+ * @see AzureCliCredential
+ */
+public class AzureCliCredentialBuilder extends AadCredentialBuilderBase<AzureCliCredentialBuilder> {
```

Comment:
What happens if the user sets the `clientId`? I can understand setting the `tenantId`.